include/opendht/network_engine.h

```diff
@@ -462,8 +464,8 @@ class NetworkEngine final {
     const Logger& DHT_LOG;
 
     NodeCache cache {};
-    std::queue<time_point> rate_limit_time {};
-    static std::mt19937 rd_device;
+    Limiter<MAX_REQUESTS_PER_SEC> rate_limiter {};
+    std::map<SockAddr, Limiter<MAX_REQUESTS_PER_SEC/8>> address_rate_limiter {};
```

Comment:
Why divide by 8? I get that 1600 requests per ip may be too high since this is presently set to be a global max, but why just divide by 8? Is it related to `TARGET_NODES` constant? May be you could comment on it.